modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -201,26 +201,20 @@ const Quirks = (editor: Editor): Quirks => {
   };
 
   /**
+   * It seems that Chrome doesn't place the caret if you click on the documentElement in iframe mode
+   * something that is very easy to do by accident so this problem is now more generic than the original issue.
+   *
+   * Original IME specific issue:
    * WebKit has a weird issue where it some times fails to properly convert keypresses to input method keystrokes.
    * The IME on Mac doesn't initialize when it doesn't fire a proper focus event.
```

Comment:
Is this still needed? It doesn't have much to do with the current code and it'll always be in git history, so it doesn't appear to be providing much detail for the current implementation.